coopr-docs/docs/source/guide/superadmin/chef-solo-automator-plugin.rst

```diff
@@ -227,16 +295,18 @@ The example coopr service definition invoking this cookbook is called "base". It
 action of type "chef-solo" and run_list ``recipe[coopr_base::default]`` (which includes ``recipe[coopr_hosts::default]``).
 Note that the community "hostsfile" cookbook is not needed in the run-list since it is declared in coopr_hosts's metadata.
 
-**coopr_service_runner**
-------------------------
+By default, it will create an ``/etc/hosts`` entry for each node of the cluster, using its private ``bind_v4`` IP.  It is possible to modify this behavior
+using the ``node['coopr_hosts']['address_types']`` attribute array. See :doc:`Macros </guide/admin/macros>` for more information on the available address types.
+
+**coopr_service_manager**
+-------------------------
 
 This cookbook comes in handy as a simple way to isolate the starting and stopping of various services within your
 cluster. It allows you to simply specify the name of a Chef service resource and an action within a Coopr service
 definition. When run, it will simply lookup the Chef service resource of the given name, regardless of which cookbook
 it is defined in, and run the given action. In the example apache-httpd service definition above, it is simply included
 in the run-list to start or stop the apache2 service defined in the apache2 community cookbook. All that is needed is
```

Comment:
i disagree with the run-list.  Chef docs use "run-list", ie http://docs.chef.io/nodes.html#about-run-lists